### PR TITLE
.travis.yml: Add exclude branch 'sils/*'

### PR DESCRIPTION
**Pull request by Flap-Py**
Turns off the travis build for branch
'sils/*' to reduce the unneeded delays.

closes https://github.com/coala/coala/issues/3516

### Checklist

- [ ] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [ ] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [ ] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [ ] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [ ] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [ ] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -77,3 +77,7 @@ script:
 
 notifications:
   email: false
+
+branches:
+  exclude:
+    - /^sils\//
```
Comment (Techievena) [inline .travis.yml]: As @sils suggested in https://github.com/coala/coala-bears/pull/1289 , you need to add a newline before EOF

**Comment by sils** [comment]
closes -> Closes

just because you need to rebase anyway, otherwise I'd have it let slip through TBH

File: .travis.yml (hunk 1 above)
Comment (sils) [inline .travis.yml]: the newline is there @Techievena  you can see it's missing by github showing a red icon at the end of the line

Comment (Techievena) [inline .travis.yml]: Sorry for misinterpreting

**Comment by Mixih** [comment]
ack e50cac0

**Comment by Mixih** [comment]
@rultor merge

**Comment by Mixih** [comment]
oops, @Flap-Py please rebase

**Comment by Mixih** [comment]
@rultor merge

**Comment by Flap-Py** [comment]
requires manual review

**Comment by sils** [comment]
and something else was merged again :/

**Comment by sils** [comment]
reack https://cdn.rawgit.com/coala/artwork/master/artwork/asciinema_demo/asciinema_demo.png

**Comment by sils** [comment]
reack 0436969

**Comment by sils** [comment]
I cherry picked your commit on https://github.com/coala/coala/pull/3563 because we have so many PRs ATM and that would end in rebase hell.

**Pull request closed without merging**